qiime2/sdk/action.py

```diff
@@ -283,45 +284,54 @@ def _build_annotations(self):
         return annotations
 
     def _build_numpydoc(self):
-        numpydoc = ""
-        numpydoc += "{}\n\n".format(self.name)
-        numpydoc += "{}\n\n".format(self.description)
+        numpydoc = []
+        numpydoc.append(textwrap.fill(self.name, width=79))
+        numpydoc.append(textwrap.fill(self.description, width=79))
 
         sig = self.signature
         params = True if len(sig.inputs) != 0 else False
```

Comment:
This comes up in a lot of places `if len(thing) != 0:` you can replace that with just `if thing:`